packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_fr_FR.json

```diff
@@ -27,6 +27,9 @@
   "pci_database_add_subnet_add": "Créer un sous-réseau",
   "pci_database_add_subnet_loading": "Chargement de sous-réseaux",
   "pci_database_add_step6_title": "Configuration du service de base de données",
+  "pci_database_add_command_description": "La commande suivante vous permet de créer votre service grâce à notre API.",
+  "pci_database_add_command_documentation_link": "Cliquez-ici pour accéder à la documentation.",
+  "pci_database_add_command_api_link": "notre API",
```

Comment:
Is it still used?

Reply:
Yeah, it's not used anymore, so I removed it :)